packages/ssz/test/unit/byType/listBasic/tree.test.ts

```diff
@@ -0,0 +1,135 @@
+import {ListBasicType, UintNumberType} from "../../../../src";
+import {runViewTestMutation, TreeMutation} from "../runViewTestMutation";
+
+const limit = 100;
+const ListN64Uint64NumberType = new ListBasicType(new UintNumberType(8, true), limit);
+
+runViewTestMutation({
+  type: new ListBasicType(new UintNumberType(8, true), limit),
+  mutations: [
+    {
+      id: "push basic",
+      valueBefore: [],
+      valueAfter: [100],
+      fn: (tv) => {
+        tv.push(100);
+      },
+    },
+    {
+      // push 2 times to increase depth and test the expand functionality
+      id: "push basic x2",
+      valueBefore: [],
+      valueAfter: [1, 2],
+      fn: (tv) => {
+        tv.push(1);
+        tv.push(2);
+      },
+    },
+    {
```

Comment:
in this test, byteLength = 8, with a chunk of 32 bytes => itemsPerChunk = 4, we should add a test to push 5 times to add 1 more chunk